pkg/server/api/trustdomain/v1/service.go

```diff
@@ -311,3 +318,14 @@ func fieldsFromRelationshipProto(proto *types.FederationRelationship, mask *type
 
 	return fields
 }
+
+func validateEndpointBundle(ctx context.Context, ds datastore.DataStore, log logrus.FieldLogger, endpointTD spiffeid.TrustDomain) {
```

Comment:
we should pass the whole SPIFFE ID and not just the trust domain so we can log appropriately below (we are currently logging the trust domain under the "endpoint_spiffe_id" label)